### DISCUSSION TITLE
Any way to autofix `react/no-unescaped-entities`?

### DISCUSSION THREAD

**Discussion by sneko** [prettier/prettier#13954] [Q&A] [answered]
Hi,

In my JSX templating, if I have text with `'` I have to convert it to `&apos;`, which is pretty annoying when taking text from mockups...

I know a solution would be to use i18n library to manage text elsewhere, but my project does not need that for now.

Is there a way Prettier could escape entities only in templating?

Thank you,

EDIT: disabling the rule does not seem like a solution since it's recommended

**Comment by duailibe** [comment, marked as answer]
Prettier will try hard to not change the contents of your code, only whitespace (see https://prettier.io/docs/en/rationale.html#correctness).

Yes, they may be equivalent when using JSX with HTML, but you may want to use JSX for other uses where `&apos;` doesn't make sense.

And it's unlikely an option would be added for this.
